code/modules/projectiles/bullet.dm

```diff
@@ -674,9 +674,22 @@ toxic - poisons
 
 	seeker
 		name = "drone-seeking grenade"
+		power = 50 //even if they don't explode, you FEEL this one
 		var/max_turn_rate = 20
 		var/type_to_seek = /obj/critter/gunbot/drone //what are we going to seek
 		precalculated = 0
+		on_hit(atom/hit, angle, var/obj/projectile/P)
+			if (P.data || prob(10)) //maybe
```

Comment:
Wouldn't this make it not hit pods properly?

Reply:
P.data is non-null if it is seeking a pod, so it'll go up to the parent proc in `/datum/projectile/bullet/autocannon` where the explosion is done.